docs/en/Host-Skill-As-Webservice.rst

```diff
@@ -0,0 +1,126 @@
+====================================
+Host a Custom Skill as a Web Service
+====================================
+
+You can build a custom skill for Alexa by implementing a web service that
+accepts requests from and sends responses to the Alexa service in the cloud.
+
+The web service must meet certain requirements to handle requests sent by Alexa
+and adhere to the Alexa Skills Kit interface standards. For more information,
+see
+`Host a Custom Skill as a Web Service <https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-a-web-service.html>`__
```

Comment:
This name is same as the tilte, which can be confusing when reading the doc, quoting itself in the doc. I know this doc is pointing to developer doc, but maybe better to rename either this or title